hbp_knowledge/proteostasis/morimoto2015.bel

```diff
@@ -73,212 +73,274 @@ DEFINE ANNOTATION TextLocation  AS URL "https://arty.scai.fraunhofer.de/artifact
 act(p(HGNC:HSF1)) -> tloc(p(HGNC:HSF1), MESH:"Cytosol", MESH:"Cell Nucleus")
+#Not sure of this?
 tscript(p(HGNC:HSF1)) -> a(MESH:"Molecular Chaperones")
+UNSET Confidence
 
 SET Evidence = "Once stress
 has been relieved, HSF1 activity is repressed through acetylation and binding to molecular
 chaperones (34, 36, 37)."
+SET Confidence = "High"
 p(HGNC:HSF1, pmod(Ac)) -| act(p(HGNC:HSF1))
 complex(p(HGNC:HSF1),a(MESH:"Molecular Chaperones")) -| act(p(HGNC:HSF1))
+UNSET Confidence
 
 SET Evidence = " IRE1 is a transmembrane protein with kinase and endoribonuclease (RNase) activity
 that senses misfolding in the ER directly, leading to autophosphorylation, oligomerization,
 and acquisition of RNase activity (8)."
+SET Confidence = "High"
 SET MeSHAnatomy = "Endoplasmic Reticulum"
 bp(GO:"response to misfolded protein") -> kin(p(HGNC:ERN1))
 kin(p(HGNC:ERN1)) -> p(HGNC:ERN1, pmod(Ph))
 p(HGNC:ERN1, pmod(Ph)) -> act(p(HGNC:ERN1), ma(GO:"ribonuclease activity"))
 UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence = "This process allows active IRE1 to cleave XBP1
 messenger RNA (mRNA), thereby generating a spliced transcript (XBP1s) that encodes a
 stable form of XBP1 that binds DNA and induces transcription of UPR target genes (8)."
 #UPR (unfolded protein response)
+SET Confidence = "High"
 act(p(HGNC:ERN1), ma(GO:"ribonuclease activity")) -> p(HGNC:XBP1)
 tscript(p(HGNC:XBP1)) -> bp(GO:"response to misfolded protein")
+UNSET Confidence
 
 SET Evidence = " In
 parallel, ER stress promotes the relocation of ATF6 from the ER membrane to the Golgi
 apparatus, where it is cleaved by SP1 and SP2 proteases."
+SET Confidence = "High"
 bp(MESH:"Endoplasmic Reticulum Stress") -> tloc(p(HGNC:ATF6), MESH:"Endoplasmic Reticulum", MESH:"Golgi Apparatus")
 SET MeSHAnatomy = "Golgi Apparatus"
 p(HGNC:PRSS1) -| p(HGNC:ATF6)
 p(HGNC:PRSS2) -| p(HGNC:ATF6)
 p(HGNC:PRSS1) -> p(HBP:"N-terminal fragment of ATF6")
 p(HGNC:PRSS2) -> p(HBP:"N-terminal fragment of ATF6")
 UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence = "The cytosolic N-terminal fragment
 of ATF6 that is generated translocates to the nucleus, binds DNA, and drives expression of a
 complementary set of UPR genes (8)."
+SET Confidence = "Low"
 tloc(p(HBP:"N-terminal fragment of ATF6"), MESH:"Cytosol", MESH:"Cell Nucleus") -> tscript(p(HBP:"N-terminal fragment of ATF6"))
 tscript(p(HBP:"N-terminal fragment of ATF6")) -> bp(GO:"response to misfolded protein")
-
+UNSET Confidence
 
 SET Evidence = " Finally, a third ER transmembrane protein, PERK,
 promotes translation of the TF ATF4 by phosphorylating the translation initiation factor
 eIF2α in response to ER stress."
 #PERK is EIF2AK3
-bp(MESH:"Endoplasmic Reticulum Stress") -> kin(p(HGNC:EIF2AK3))
-kin(p(HGNC:EIF2AK3)) -> p(HGNC:EIF2S1, pmod(Ph))
+SET Confidence = "Low"
+bp(MESH:"Endoplasmic Reticulum Stress") -> p(HGNC:EIF2AK3)
+act(p(HGNC:EIF2AK3)) -> p(HGNC:EIF2S1, pmod(Ph))
 p(HGNC:EIF2S1, pmod(Ph)) -> tscript(p(HGNC:EIF2S1))
```

Comment:
why in confidence low here?
please exchange transcriptional activity of EIF2S1 by translational activity

Reply:
@sgebel the statements were marked "Low" as i wasn't sure about the use of tscript BEL statements. Maybe we can discuss this tomorrow.

Reply:
please code translational activity using ma(GO:"translation regulator activity")